README.md

```diff
@@ -69,14 +74,51 @@ def ceAsync =
   }
 ```
 
+## Other typeclasses
+
+There are many other typeclasses and useful conversions that this library provides implementations for:
+* See `zio/interop/cats.scala` file to see all available typeclass implementations for the Cats Effect 3 typeclasses
+* See `zio/stream/interop/cats.scala` for ZStream typeclass implementations
+* See `zio/stream/interop/FS2StreamSyntax.scala` for FS2 <-> ZStream conversions
+
+## Easier imports (at a cost)
+
+In the examples above, we had to bring the `Runtime[Clocking with Blocking]` via the `ZIO.runtime` combinator. This may
+not be ideal since everywhere you use these typeclasses, you will now be required to feed in the `Runtime`. 
+For example, with FS2: 
+
+```scala
+def example(implicit rts: Runtime[Clock with Blocking]): Task[Unit] =
+  fs2.Stream
+    .awakeDelay[Task](10.seconds) // this type annotation is mandatory
+    .evalTap(in => cats.effect.std.Console.make[Task].println(s"Hello $in"))
+    .compile
+    .drain
+```
+
+Rather than requiring the runtime implicit, we can add an import (if we don't mind depending on `Runtime.default`):
+```scala
+import zio.interop.catz._
+import zio.interop.implicits._
```

Comment:
Sorry! just this one place needs to be updated and that'll be it 🙏

Reply:
oops this is my bad! i will fix now sorry for the trouble!